docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -35,18 +35,24 @@ code: |
       not_finishing_end
   else:
     progressor.increment()  # 1
-  
+
   progressor.increment()  # 2
-  
+
   needs_dockets = False
+  # If user doesn't answer yes for docket sheets, show user the docket sheet description
   if ( not have_docket_copies_already is True ):
     progressor.increment()  # 3
     needs_dockets = True
-    if ( have_docket_copies_already is None ):  # They don't know
-      about_docket_sheets
+    # If user wants to see docket sheet instructions
     if ( wants_dockets_first is True ):
-      progressor.increment()  # 4
-      not_finishing_end  # end
+      # User has not seen docket sheet instructions yet
+      if ( not about_docket_sheets is True ):
```

Comment:
I find the `if ( not x is True )` construction really confusing, though I see it's also used above. I would generally prefer `if ( about_docket_sheets )` or `if ( about_docket_sheets in (False, None) )` if you want to be really explicit